src/handler.js

```diff
@@ -20,12 +21,12 @@ const writeError = (res, status, code, message) => {
   res.end(JSON.stringify({ code, message }));
 };
 
-const createProxy = config => {
+const createProxy = apiKey => {
   const proxy = httpProxy.createProxyServer({
     changeOrigin: true,
     headers: {
       Accept: 'application/json',
-      Authorization: `Bearer ${config.airtableApiKey}`,
+      Authorization: `Bearer ${apiKey}`,
```

Comment:
This was using `config.airtableApiKey`, but `config` was just the string of the API key. Fixed that and now the functionality checks out.